test/assets/assets_common.json

```diff
@@ -0,0 +1,4 @@
+{
+    "assets": [
```

Comment:
what is the reason for this empty file?

Reply:
I autocleaned that via some sed command so didnt notice its empty already.
removed